src/engraving/libmscore/score.cpp

```diff
@@ -3320,8 +3320,15 @@ void Score::deselect(EngravingItem* el)
 
 void Score::select(EngravingItem* e, SelectType type, staff_idx_t staffIdx)
 {
-    // Move the playhead to the selected element's preferred play position.
     if (e) {
+        if (e->type() == ElementType::TEXT) {
```

Comment:
We have a special flag for such situations. You should use it instead of adding local solutions to this method

<img width="617" alt="Screenshot 2022-09-30 at 11 09 39" src="https://user-images.githubusercontent.com/67867444/193223420-d21faf50-296c-4500-9695-aefa4bac26fc.png">